dev-packages/e2e-tests/test-applications/vue-3/tests/performance.test.ts

```diff
@@ -122,3 +122,101 @@ test('sends a pageload transaction with a route name as transaction name if avai
     },
   });
 });
+
+test('sends a lifecycle span for the tracked HomeView component - with `<>`', async ({ page }) => {
+  const transactionPromise = waitForTransaction('vue-3', async transactionEvent => {
+    return !!transactionEvent?.transaction && transactionEvent.contexts?.trace?.op === 'pageload';
+  });
+
+  await page.goto(`/`);
+
+  const rootSpan = await transactionPromise;
+
+  expect(rootSpan).toMatchObject({
+    contexts: {
+      trace: {
+        data: {
+          'sentry.source': 'route',
+          'sentry.origin': 'auto.pageload.vue',
+          'sentry.op': 'pageload',
+        },
+        op: 'pageload',
+        origin: 'auto.pageload.vue',
+      },
+    },
+    spans: expect.arrayContaining([
+      // enabled by default
+      expect.objectContaining({
+        data: {
+          'sentry.op': 'ui.vue.render',
+          'sentry.origin': 'auto.ui.vue',
+        },
+        description: 'Application Render',
+        op: 'ui.vue.render',
+        origin: 'auto.ui.vue',
+      }),
+      // enabled by default
+      expect.objectContaining({
+        data: {
+          'sentry.op': 'ui.vue.mount',
+          'sentry.origin': 'auto.ui.vue',
+        },
+        description: 'Vue <<Root>>',
+        op: 'ui.vue.mount',
+        origin: 'auto.ui.vue',
+      }),
+      expect.objectContaining({
+        data: {
+          'sentry.op': 'ui.vue.mount',
+          'sentry.origin': 'auto.ui.vue',
+        },
+        description: 'Vue <<HomeView>>',
```

Comment:
Just making sure, are the double brackets intended? They're formatted [here](https://github.com/getsentry/sentry-javascript/blob/a8e5f593b3341f90cf9aafaaa1f6eb1e0da7545e/packages/vue/src/tracing.ts#L112).

Reply:
Good observation! I don't think they're intended. Let's remove them.